tests/bin/get_att.c

```diff
@@ -0,0 +1,15 @@
+#include "libc.h"
+#include <errno.h>
+
+int main(void) {
+    // TODO: Good buffer length?
+    unsigned char nonce[512];
+    unsigned char buf[512];
+    size_t technology;
+
+    ssize_t size = get_att(nonce, sizeof(nonce), buf, sizeof(buf), &technology);
+
+    // TODO: Currently this should return ENOSYS. However, this will change
+    // as get_att() is implemented for each technology.
+    return (size != -1 || errno != ENOSYS);
```

Comment:
I find this logic hard to grok without thinking. I think we should do this instead, since it is much more obvious.

```suggestion
    return !(size == -1 && errno == ENOSYS);
```

The logic inside the parentheses expresses our success condition. The exclamation point outside the parentheses inverts the success condition into an exit status.